src/main/routes/action.resource.js

```diff
@@ -7,56 +7,46 @@ const router = express.Router();
 
 export default function(QueryHandler) {
     router.put('/', async(req, res) => {
-        if (!req.body.scenario) res.send({ status: 400, error: 'No scenario specidied' });
         const idScenario = req.body.scenario;
         const action = req.body;
-
-        const scenario = await QueryHandler.get(idScenario);
-        if (scenario) {
+        var response = {};
+        try {
+            const scenario = await QueryHandler.get(idScenario);
             const verifier = new VerifierInterface(scenario, 'action');
-
-            var response = verifier.check(action);
-            if (response == true) {
+            if (verifier.check(action) == true) {
                 const handler = new ScenarioHandler(scenario);
                 delete action.scenario;
-                console.log(handler.scenario);
                 handler.pushAttribute(action, 'actions');
-                // console.log(handler.scenario);
                 await QueryHandler.set(idScenario, handler.scenario);
-                res.send({ code: 202, scenario: handler.scenario });
+                response = { code: 202, scenario: handler.scenario };
+            } else {
+                response = { code: 400, error: verifier.check(action) };
```

Comment:
This else is necessary? i see the same response inside the catch, so if it's not necessary please remove this else.

Reply:
Yes, because the catch is an error generated in db, that may be a bad id or an scenario that does not exist. The error in the else is that maybe the actor in the action does not exist in the actors of the scenario

Reply:
Oh ok, so i will approve